package.json

```diff
@@ -13,28 +13,29 @@
     "gettimeofday"
   ],
   "engines": {
-    "node": ">= 4.0.0"
+    "node": ">= 14.13.0"
```

Comment:
FYI this was a semver-major change. @wadey is there a plan to relax the engines requirement back to `"node": ">= 4.0.0"` for the remainder of the 3.x release lifecycle? (or perhaps even just node 12, since that was the most recently expired LTS, or node 10 before it)?